dxr/plugins/xpidl/visitor.py

```diff
@@ -0,0 +1,244 @@
+from cStringIO import StringIO
+from itertools import ifilter
+from os.path import relpath, join, basename, dirname, exists
+
+from flask import url_for
+from xpidl.header import idl_basename, header, include, jsvalue_include, \
```

Comment:
The backslash has fallen out of fashion. Everybody uses parens to wrap these days.